index.html

```diff
@@ -1312,6 +1328,10 @@ <h4>Mandatory fields</h4>
             <span class="rfc2119-assertion" id="profile-5.1.5.1-thing-action-affordances-mandatory-fields-1">
             The following fields MUST be contained in an
             action element of an <a>Core Profile</a> compliant TD:
+	     <em>
+	     FIX: this should have an table assertion per row, and this should be informative
+	     or a parent assertion.
+	     </em>
```

Comment:
I had to restructure this to avoid having assertions nested inside assertions.  So now there is an assertion for each row (asserting the element must be present) and two SEPARATE assertions after the table indicating that these must use simple types.  There are some technical issues with these assertions though (dataSchema should be DataSchema?  What is a simple type?  If DataSchema can only have simple types, then why is objectSchema relevant?) but let's deal with those separately.

Reply:
Sigh... I'll have to update my tool to treat these correctly.  In the TD there was a column indicating whether a row element was mandatory or optional, but the implication here is that all of these are mandatory, and there is no such column.  A fairly simple tweak, though...